app/assets/javascripts/admin/spree/orders/variant_autocomplete.js.erb

```diff
@@ -154,3 +157,20 @@ addVariantFromStockLocation = function() {
   }
   return 1
 }
+
+showAlert = function(message) {
```

Comment:
This is defining the method globally, right?

I'm okay with this for now but we may want to share it with other parts of the application at some point in the future and then it could be good to use a clearer name. Maybe `ofnAlert`? No need to change it now, I think.

Reply:
I kept these methods in the same file because I was getting some errors in the CI when I tried to move them elsewhere even though the code worked fine in the browser.